addon/components/nypr-accounts/membership-card/giving-history.js

```diff
@@ -13,7 +13,9 @@ export default Component.extend({
   sortedSustainingPledges: computed.filterBy(
     "sortedPledges",
     "orderType",
-    "sustainer"
+    "sustainer",
+    "isPayment",
+    true
```

Comment:
Unless I'm mistaken, you can't overload the `filterBy` function here. You could just do the regular `.filter` and use a callback that filters by both of those values.

Reply:
doh - for some reason I was thinking the filters could be multiple. I added a sortedSustainingPayments which is based off of sortedSustainingPledges but filters on the isPayment field (that is active and accurate from nF). This should only change the Giving History view and not the MY DONATION STATUS pane.